mlflow/utils/autologging_utils/__init__.py

```diff
@@ -43,6 +43,9 @@
 )
 _AUTOLOGGING_TEST_MODE_ENV_VAR = "MLFLOW_AUTOLOGGING_TESTING"
 
+# Flag indicating whether autologging is globally disabled for all integrations.
+AUTOLOGGING_GLOBALLY_DISABLED = False
```

Comment:
Nit: Can we make this "internal" since it doesn't need to be referenced outside of the file in which it's defined?

```suggestion
_AUTOLOGGING_GLOBALLY_DISABLED = False
```

Reply:
I thought that as well, but it is primarily used in `safety.py`. Is it still ok to reference an internal variable like that since it (should be) the only usage?

Reply:
I think that's okay :)